start.sh

```diff
@@ -79,10 +79,22 @@ echo "== Connecting to notebook =="
 print_logs
 
 echo
-
 instruction_get_logs
-echo
-echo "== Instructions =="
-echo "1. Password, output, and error printed to this terminal? Look at logs (see instruction above)"
-echo "2. Browser: http://sh-02-21.int:$PORT/ -> http://localhost:$PORT/..."
-echo "3. To end session: bash end.sh ${NAME}"
+
+if [[ "${RESOURCE}" == "sherlock" ]]
+then
+  echo
```

Comment:
This is a bit redundant and should use variables instead

Reply:
Fixed as per 6aa907f

Reply:
This is almost spot on! We still don't want SHERLOCK as a hard coded variable. See if you can add this logic to the setup.sh with a variable.